submissions/2025-03-31-OpenFF-Cresset-Additional-Coverage-Hessian-v4.0/README.md

```diff
@@ -0,0 +1,50 @@
+# OpenFF Cresset Additional Coverage Hessian v4.0
+
+## Description
+
+Hessian single points for the final molecules in the [OpenFF Cresset Additional Coverage Optimizations v4.0 dataset](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2025-03-06-OpenFF-Cresset-Additional-Coverage-Optimizations-v4.0) at the B3LYP-D3BJ/DZVP level of theory.
```

Comment:
According to v4 standards this should also contain:
 - Intended usage of the dataset
 - Elements covered
 - Charges covered
 - Min, mean, and max molecular weight of the molecules in the dataset

Reply:
Thanks for catching that, updated the description in the notebook, dataset, and readme!